package.json

```diff
@@ -49,5 +49,8 @@
     "prettier-eslint": "^15.0.1",
     "prettier-eslint-cli": "^7.1.0",
     "prettier-plugin-tailwindcss": "^0.2.2"
+  },
+  "dependencies": {
+    "react-infinite-scroll-component": "^6.1.0"
```

Comment:
This is for the front end yeah? This package should be added to the client/package.json then. It'll still be installed and added to the package-lock but it'll be specific to the client workspace (that's why you don't see the other deps in this file)